spec/rails_helper.rb

```diff
@@ -21,7 +21,7 @@
 # directory. Alternatively, in the individual `*_spec.rb` files, manually
 # require only the support files necessary.
 #
-Dir[Rails.root.join("spec/support/**/*.rb")].sort.each { |f| require f }
+Dir[Rails.root.join("spec/support/**/*.rb")].each { |f| require f }
```

Comment:
What does this do?

Reply:
This line ensures that test helper functions are correctly imported when running tests. The `sort` step was redundant as the order in which they're imported doesn't matter. I don't know what prompted removing it here.